assets/javascripts/card-form-customization.js

```diff
@@ -91,3 +109,4 @@ document.getElementById('omise-modal').addEventListener('click', (event) => {
 
 setDesignFormValues();
 handleColorInputChanges();
+handleFontChange();
```

Comment:
The line above uses "Changes" instead of "Change". In the interests of consistency, should we make this `handleFontChange()` or rename `handleColorInputChanges()` to `handleColorInputChange()`?

Reply:
I didn't end `handleFontChange` with `s` because it touches only one field where as `handleColorInputChanges` touches multiple color input fields like text color, border color, label color and so on. Do you still think it would be better to end `handleFontChange` with `s`?

Reply:
I do prefer `handleFontChange`. I would also prefer `handleColorInputChange` as I'm not sure that the fact that it touches multiple fields is communicated by "changes" over "change", but we can discuss in a separate PR.